rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleTransactionBase.java

```diff
@@ -106,6 +106,16 @@ public String sendRawTransaction(String rawData) {
 				throw invalidParamError("Missing parameter, gasPrice, gas or value");
 			}
 
+			Account senderAccount = this.wallet.getAccount(tx.getSender());
+
+			if (senderAccount == null) {
+				throw RskJsonRpcRequestException.invalidParamError(TransactionArgumentsUtil.ERR_COULD_NOT_FIND_ACCOUNT + tx.getSender().toHexString());
+			}
```

Comment:
we don't need this particular sender account check in the `sendRawTransaction()` method, as the sender account doesn't really need to be part of the `wallet` in this context.

Reply:
Done